TradeItIosTicketSDK2/TradeItQuotePresenter.swift

```diff
@@ -25,7 +25,12 @@ class TradeItQuotePresenter: NSObject {
         var result = ""
 
         if let change = change {
-            result = NumberFormatter.formatCurrency(change, currencyCode: self.currencyCode)
+            result = NumberFormatter.formatCurrency(
```

Comment:
What does this do for non-CIMB? Does it show 2 or 3 decimal places?

Reply:
2 if you put 2, 3 if you put 3